pyuvdata/uvdata.py

```diff
@@ -1074,10 +1154,17 @@ def reorder_pols(self, order='AIPS', run_check=True, check_extra=True,
 
     def order_pols(self, order='AIPS'):
         """
-        Soon to be deprecated method, now just calls reorder_pols.
+        Will be deprecated in version 1.5, now just calls reorder_pols.
 
-        Args:
-            order: string, either 'CASA' or 'AIPS'. Default='AIPS'
+        Parameters
+        ----------
+        order : str
+            either 'CASA' or 'AIPS'.
+
+        Warns
+        -----
+        DeprecationWarning
+            Always, because this method will be deprecated in version 1.5
```

Comment:
Should this have a Raises section as well? I guess not because it's technically not this function raising the error.

Reply:
I can add one.